### PR TITLE
Introduced component state #13

### PR DESCRIPTION
**Pull request by tishayaem**

### CODE REVIEW

File: src/components/Details.js [hunk 1]
```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 
 export default class Details extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
+    }
+
+    componentDidMount() {
+        this.timerID = setInterval(
```
Comment (rmallols) [inline src/components/Details.js]: 2 things here:
1. You're using `setInterval`, which means the function will run, infinitely, every 3 seconds. We want to run it just once, using `setTimeout` instead
2. You are saving a variable `timerID`, but you aren't using it at all. Hence, I believe we don't need it.

File: src/components/Details.js [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+            () => this.newMessage(),
```
Comment (rmallols) [inline src/components/Details.js]: I think we need an additional indentation here (try with `option + shift + f`)

File: src/components/Gallery.js [hunk 3]
```diff
@@ -10,8 +9,9 @@ export default class Gallery extends React.Component {
                     {
                         movieDB.map(function (movie) {
                             return <Covers image={movie.image} title={movie.title} id={movie.id} type={movie.type} />
-                        });  
+                        })
                     }
+
```
Comment (rmallols) [inline src/components/Gallery.js]: Empty line here, could you please remove it? :)

File: src/css/App.css [hunk 4]
```diff
@@ -53,7 +53,7 @@ h1 {
   box-sizing: border-box;
 }
 h2 {
-  background-color: blue;
+  background-color: lightcoral;
```
Comment (rmallols) [inline src/css/App.css:56]: Nice :)

File: src/components/Details.js [hunk 5]
```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 
 export default class Details extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
```
Comment (rmallols) [inline src/components/Details.js]: I'd suggest you to always add an empty space separating curly brackets, for readability purposes

File: src/components/Details.js (hunk 2 above)
Comment (rmallols) [inline src/components/Details.js]: I'd be great to keep a more explicit, action-oriented naming conventions to describe functions:
1. Set the action you are going to perform (e.g. `setWhatever`, `returnSomething`, etc).
2. Describe the purpose of the action (you are setting a `Coming Soon` message, so the name of the action should reflect that, somehow.

File: src/css/App.css (hunk 4 above)
Comment (tishayaem) [reply src/css/App.css:56]: makes all the difference)

**Review by rmallols — CHANGES_REQUESTED** [review]
Looking good - just some subtle changes requested

File: src/components/Details.js [hunk 6]
```diff
@@ -3,24 +3,24 @@ import React from 'react';
...
     constructor(props) {
         super(props);
-        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
+        this.state = { message: 'Hello, this will be the details page for each Movie & TV show' };
     }
 
     componentDidMount() {
         setTimeout(
-            () => this.newMessage(),
+            () => this.comingSoonMessage(),
```
Comment (rmallols) [inline src/components/Details.js]: Subtle thing! Could you rename it to `setComingSoonMessage`? That would help to describe the action

**Review by rmallols — CHANGES_REQUESTED** [review]
Almost there :)

**Review by rmallols — APPROVED** [review]
Awesome, cheers :)

**Pull request merged**